src/ring_theory/polynomial/hermite/basic.lean

```diff
@@ -127,4 +129,59 @@ end
 
 end coeff
 
+section coeff_explicit
+
+open_locale nat
+
+/- Because of `coeff_hermite_of_odd_add`, every nonzero coefficient is described as follows.  -/
+lemma coeff_hermite_explicit :
+  ∀ (n k : ℕ), coeff (hermite (2 * n + k)) k = (-1)^n * (2 * n - 1)‼ * nat.choose (2 * n + k) k
```

Comment:
I think it's probably worth having something like
```lean
  even (m + k) -> coeff (hermite m) k = (-1)^((m - k) / 2) * (m + 1 - k)‼ * nat.choose m k
```
as well, proved in terms of your current version; this provides a nice complement to `coeff_hermite_of_odd_add`

Reply:
OK, done.